src/proto/BitcoinGold.proto

```diff
@@ -0,0 +1,11 @@
+// Copyright © 2017-2020 Trust Wallet.
```

Comment:
remove copy/paste code, I don't think we need this

Reply:
This proto file is not needed

Reply:
removed!